### DISCUSSION TITLE
How to restore pop up

### DISCUSSION THREAD

**Discussion by jalanh11640** [devcontainers/community#138] [Q&A]
When I opened my project that contains a devcontainer.json, I used to get a pop up that says something to the effect of "Folder contains a Dev Container configuration file.  Rebuild and open in container?"  I'm no longer seeing that popup.  My guess is I inadvertently clicked the "Don't show again button."

How do I get that pop up back?

Thanks!

**Comment by samruddhikhandale** [comment]
In Visual Studio Code, you can reset the prompt for reopening in a container by following these steps:

- Open the Command Palette with Cmd+Shift+P on macOS or Ctrl+Shift+P on Windows/Linux.
- Type Preferences: Open User Settings and select it.
- In the search bar of the Settings tab, type remote.containers.showReloadPrompt and make sure it's checked. This should re-enable the prompt for reopening in a container when a devcontainer.json file is detected in your project.

If you still don't see the prompt, try reloading Visual Studio Code. If the problem persists, check your devcontainer.json file for any errors that might prevent VS Code from recognizing it as a valid Dev Container configuration file.

If the suggested steps doesn't help, then can can you open an issue in https://github.com/microsoft/vscode/issues? The VS Code team should be able to help you provide guidance. Thanks!
